README.md

```diff
@@ -108,7 +108,7 @@ await fastify.ready()
  | transformStaticCSP   | undefined        | Synchronous function to transform CSP header for static resources if the header has been previously set.                  |
  | transformSpecification     | undefined        | Synchronous function to transform the swagger document.                                                                   |
  | transformSpecificationClone| true             | Provide a deepcloned swaggerObject to transformSpecification                                                                    |
- | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md). Must be literal values, see [#5710](https://github.com/swagger-api/swagger-ui/issues/5710).                                                                                                            |
+ | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md).                                                                                                           |
```

Comment:
Please add a detail on `function` should be self contained which means any variables outside the `function` would not works.

Reply:
Actually the swagger UI element is bound to window.ui. so you could access that from the function.

```suggestion
 | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md). Functions should be self contained, as variables outside the function are potentially not working. You can access the swagger-ui element with `ui` as it is assigned to `window.ui`.                                                                                                   |
```

Like th8s?

Reply:
> Like this?

`potentially not working` is would give a strong stance here `not working`.

Reply:
```suggestion
 | uiConfig             | {}               | Configuration options for [Swagger UI](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md). Functions should be self contained, as variables outside the functions are  not working. You can access the swagger-ui element with `ui` as it is assigned to `window.ui`.                                                                                                   |
```

Reply:
Well I had a good laugh about the example. But what about accessing global scope window? :D I was thinking of that and not thinking to access fastify backend vaiables :D

Reply:
Adapted it